<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>615</width>
    <height>390</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Daylight - 0.0.1</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>2</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>2</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QGraphicsView" name="graphicsView">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
        <horstretch>20</horstretch>
        <verstretch>10</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustToContents</enum>
      </property>
      <property name="backgroundBrush">
       <brush brushstyle="NoBrush">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </property>
      <property name="sceneRect">
       <rectf>
        <x>0.000000000000000</x>
        <y>0.000000000000000</y>
        <width>0.000000000000000</width>
        <height>0.000000000000000</height>
       </rectf>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>615</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionPreferences"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuCalendars">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionShowYear"/>
    <addaction name="actionShowMonth"/>
    <addaction name="actionShow3Weeks"/>
    <addaction name="actionShowWeek"/>
    <addaction name="actionShowDay"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuAppointments">
    <property name="title">
     <string>Appointments</string>
    </property>
    <addaction name="actionAddUserCalendar"/>
    <addaction name="actionCalendarManager"/>
    <addaction name="separator"/>
    <addaction name="actionAddAppointment"/>
   </widget>
   <widget class="QMenu" name="menuNavigation">
    <property name="title">
     <string>Navigation</string>
    </property>
    <addaction name="actionGotoToday"/>
    <addaction name="actionPrevDate"/>
    <addaction name="actionNextDate"/>
    <addaction name="actionShowHideNavigationDlg"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuCalendars"/>
   <addaction name="menuAppointments"/>
   <addaction name="menuNavigation"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::TopToolBarArea</set>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionShowYear"/>
   <addaction name="actionShowMonth"/>
   <addaction name="actionShowWeek"/>
   <addaction name="actionShowDay"/>
   <addaction name="separator"/>
   <addaction name="actionPrevDate"/>
   <addaction name="actionNextDate"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
  </widget>
  <action name="actionAddUserCalendar">
   <property name="text">
    <string>Add Calendar</string>
   </property>
   <property name="toolTip">
    <string>Add a new Calendar </string>
   </property>
  </action>
  <action name="actionShowYear">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Year</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+1</string>
   </property>
  </action>
  <action name="actionShowMonth">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Month</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+2</string>
   </property>
  </action>
  <action name="actionShow3Weeks">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>3 Weeks</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+3</string>
   </property>
  </action>
  <action name="actionShowDay">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Day</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+5</string>
   </property>
  </action>
  <action name="actionGotoToday">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Today</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Preferences</string>
   </property>
   <property name="toolTip">
    <string>preferences</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionCalendarManager">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Calendar Manager...</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::WidgetShortcut</enum>
   </property>
  </action>
  <action name="actionAddAppointment">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Add Appointment...</string>
   </property>
  </action>
  <action name="actionNextDate">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../daylight.qrc">
     <normaloff>:/icons/right.png</normaloff>:/icons/right.png</iconset>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
   <property name="toolTip">
    <string>next day, year, month</string>
   </property>
   <property name="shortcut">
    <string>Right</string>
   </property>
  </action>
  <action name="actionPrevDate">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../daylight.qrc">
     <normaloff>:/icons/left.png</normaloff>:/icons/left.png</iconset>
   </property>
   <property name="text">
    <string>Previous</string>
   </property>
   <property name="toolTip">
    <string>previous day, month, year</string>
   </property>
   <property name="shortcut">
    <string>Left</string>
   </property>
  </action>
  <action name="actionShowHideNavigationDlg">
   <property name="text">
    <string>Navigation Dialog</string>
   </property>
   <property name="toolTip">
    <string>Navigation</string>
   </property>
  </action>
  <action name="actionShowWeek">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Week</string>
   </property>
   <property name="toolTip">
    <string>shows a week</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+4</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../daylight.qrc"/>
 </resources>
 <connections/>
</ui>
